cmd/wksctl/profile/disable/disable.go

```diff
@@ -0,0 +1,96 @@
+package disable
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/wksctl/cmd/wksctl/profile/constants"
+	"github.com/weaveworks/wksctl/pkg/git"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "disable",
+	Short: "disable profile",
+	Args:  profileDisableArgs,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return profileDisable(profileDisableParams)
+	},
+	SilenceUsage: true,
+}
+
+type profileDisableFlags struct {
+	gitUrl     string
+	push       bool
+	profileDir string
+}
+
+var profileDisableParams profileDisableFlags
+
+func init() {
+	Cmd.Flags().StringVar(&profileDisableParams.profileDir, "profile-dir", "profiles", "specify a directory for storing profiles")
+	Cmd.Flags().StringVar(&profileDisableParams.gitUrl, "git-url", "", "enable profile from the Git URL")
+	Cmd.Flags().BoolVar(&profileDisableParams.push, "push", true, "auto push after disable the profile")
+}
+
+func profileDisableArgs(cmd *cobra.Command, args []string) error {
+	if len(args) != 0 {
+		return errors.New("profile disable does not require any argument")
+	}
+	return nil
+}
+
+func profileDisable(params profileDisableFlags) error {
+	repoUrl := params.gitUrl
+	if repoUrl == "" {
+		return errors.New("Git URL of the profile must be specified")
+	}
+
+	if repoUrl == constants.AppDevAlias {
+		repoUrl = constants.AppDevRepoURL
+	}
+
+	if git.IsGitURL(repoUrl) == false {
+		return errors.New("Repository is not a Git URL")
+	}
+
+	hostName, repoName, err := git.HostAndRepoPath(repoUrl)
+	if err != nil {
+		return err
+	}
+
+	clonePath := path.Join(params.profileDir, hostName, repoName)
```

Comment:
`clonePath` doesn't seem right if we don't clone anything here :) how about `profilePath`?
suggestion: rename